boxtree/tools.py

```diff
@@ -277,28 +273,27 @@ def get_3d_knl(dtype):
         raise NotImplementedError
 
 
-def make_rotated_uniform_particle_array(queue, nparticles, dims, dtype, seed=15):
+def make_rotated_uniform_particle_array(actx, nparticles, dims, dtype, seed=15):
     raise NotImplementedError
 
 # }}}
 
 
-def particle_array_to_host(parray):
-    return np.array([x.get() for x in parray], order="F").T
+def particle_array_to_host(actx, particles):
+    return np.array([actx.to_numpy(x) for x in particles], order="F").T
 
 
 # {{{ host/device data storage
 
 class DeviceDataRecord(Record):
```

Comment:
Given that the PR narrative describes getting rid of this, how come it's still here?